corehq/apps/reports/filters/case_list.py

```diff
@@ -16,13 +18,22 @@ def sharing_group_tuple(self, g):
     @property
     @memoized
     def static_options(self):
-        options = super(CaseListFilterUtils, self).static_options
-        # replace [Active Mobile Workers] and [Deactivated Mobile Workers] with case-list-specific options
-        assert options[0][0] == "t__0"
-        return [
+        # Options to show in the filters are defined here
+        hq_user_type_options = [HQUserType.DEACTIVATED, HQUserType.DEMO_USER,
+                   HQUserType.ADMIN, HQUserType.WEB, HQUserType.UNKNOWN]
+        if Domain.get_by_name(self.domain).commtrack_enabled:
+            hq_user_type_options.append(HQUserType.COMMTRACK)
+
+        # The static options themselves are created here. The options that are specific to the case list are added
+        # first
+        filters_to_display = [
             ("all_data", _("[All Data]")),
             ('project_data', _("[Project Data]"))
-        ] + options[1:]
+        ]
+        for option in hq_user_type_options:
+            filters_to_display.append(super(CaseListFilterUtils, self).create_static_option(option))
```

Comment:
I don't think you need the `super` business here - you can just reference `self.create_static_option`.  You could also do this in one line with
```python
filters_to_display.extend(map(self.create_static_option, hq_user_type_options))
```
or
```python
filters_to_display.extend(self.create_static_option(option) for option in hq_user_type_options)
```